README.md

```diff
@@ -1,84 +1,44 @@
-# Ballerina Twilio Endpoint
+# Twilio Connector
 
-[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS, Voice & Messaging applications on 
-an API built for global scale.
-
-### Why would you use a Ballerina endpoint for Twilio
-
-Ballerina Twilio endpoint allows you to access the [Twilio REST API](https://www.twilio.com/docs/api) and perform 
-actions like sending a simple text message, making a voice call etc.
+Twilio connector provides a Ballerina API to access the [Twilio REST API](https://www.twilio.com/docs/api).
 
 ## Compatibility
 
-| Language Version           | Endpoint Version    | Twilio Basic API Version | Twilio Authy API Version |
-| -------------------------- | ------------------- | ------------------------ | ------------------------ |
-| 0.970.0-beta1-SNAPSHOT     | 0.5.10              | 2010-04-01               | v1                       |
-
-##### Prerequisites
-
-1. Download the ballerina [distribution](https://ballerinalang.org/downloads/).
+| Ballerina Language Version  | Twilio API Version | Twilio Basic API Version | Twilio Authy API Version |
+| :--------------------------:|:------------------:|:------------------------:|:------------------------:|
+| 0.970.0-beta2               | 0.8.0              | 2010-04-01               | v1                       |
```

Comment:
Please remove the first column.

Reply:
Since this is the root README.md, we will keep this info as discussed.